package.json

```diff
@@ -51,6 +51,7 @@
     "dayjs": "^1.11.7",
     "expo": "^46.0.0",
     "expo-av": "~12.0.4",
+    "expo-blur": "^12.2.2",
```

Comment:
For now, a simple white background will be fine if react native hates glassmorphism.